node-bb/README.md

```diff
@@ -0,0 +1,169 @@
+# TripleA Forums
+
+The software that powers the TripleA Forums is [NodeBB](https://github.com/NodeBB/NodeBB). 
+This document is instructions on how to build and run the Docker container of the forums using an 
+[official Docker container of NodeBB](https://hub.docker.com/r/nodebb/docker) as the base image.
+
+### Building
```

Comment:
Codacy found an issue: [[heading-increment] Heading levels should increment by one level at a time](https://app.codacy.com/gh/triplea-game/triplea/pullRequest?prid=6342386)